resources/views/dashboard.blade.php

```diff
@@ -9,30 +9,30 @@
-                          <select name="expType" class="form-control">
-                                  <option value="Utilities">Utilities</option>
-                                  <option value="Groceries">Groceries</option>
-                                  <option value="Household">Household</option>
-                                  <option value="Rent">Rent/Mortgage</option>
-                                  <option value="Loan">Loan</option>
-                                  <option value="Restaurants">Restaurants</option>
-                                  <option value="Active">Active</option>
-                                  <option value="Clothing">Clothing</option>
-                                  <option value="Entertainment">Entertainment</option>
-                                  <option value="Hobbies">Hobbies</option>
-                                  <option value="Work">Work</option>
-                                  <option value="Insurance">Insurance</option>
-                                  <option value="Savings">Savings</option>
-                          </select>
-                          <label class="control-label">Date</label>
-                          <input type="date" name="expDate" class="form-control required">
-                          <label class="control-label">Amount</label>
-                          <input type="text" name="expAmount" class="form-control required">
-                          <label class="control-label">Comments</label>
-                          <input type="textarea" name="expComments" class="form-control">
+                  <form id="expForm" action="/{{ Auth::user()->username }}/add" method="POST" class="form-horizontal">
+                      <input type="hidden" name="_token" value="{{ csrf_token() }}">
+                      <label class="control-label">Type</label>
+                      <select name="expType" class="form-control">
+                            <option value="Utilities">Utilities</option>
+                            <option value="Groceries">Groceries</option>
+                            <option value="Household">Household</option>
+                            <option value="Rent">Rent/Mortgage</option>
+                            <option value="Loan">Loan</option>
+                            <option value="Restaurants">Restaurants</option>
+                            <option value="Active">Active</option>
+                            <option value="Clothing">Clothing</option>
+                            <option value="Entertainment">Entertainment</option>
+                            <option value="Hobbies">Hobbies</option>
+                            <option value="Work">Work</option>
+                            <option value="Insurance">Insurance</option>
+                            <option value="Savings">Savings</option>
+                      </select>
+                      <label class="control-label">Date</label>
```

Comment:
We should display errors in case theyre present. Since that is front-end task, can you just put a TODO comment so that front-end people can take note of it?